data/actor-inspector/factories.js

```diff
@@ -56,22 +62,31 @@ var FactoryTable = React.createClass({
  * TODO docs
  */
 var FactoryList = React.createClass({
+  getInitialState: function() {
+    return {
+      packet: [
+        {factories: {}},
+        {factories: {}}
```

Comment:
@Honza they are the empty```main``` and ```child``` factories, but it's probably better to if I turn it into a more explicit:

```
return {
  main: {},
  child: {}
}
```

I'm going to update this pull request accordingly